### DISCUSSION TITLE
Nuxt Fonts Not Loading on Some Machines

### DISCUSSION THREAD

**Discussion by cavalluccijohann** [community/community#133523] [Programming Help]
### Body

Hey,

I've created a website with Nuxt and I'm using @nuxt/fonts, but on some machines, the fonts are not loading. I don't think it's a browser issue. Has anyone else encountered this problem?

Thank you :)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mOnteySZEF** [comment]
Jol jol _cavalluccijohann_,
First, try clearing the browser cache on the machines where it’s not working. Also, double-check the font source in your `@nuxt/fonts` setup to make sure it's correct. Look for any console errors that could give clues about what's wrong. Lastly, ensure all your dependencies are up to date. **Good luck yo**
